examples/datasets/plot_iris_dataset.py

```diff
@@ -35,6 +38,10 @@
 x_min, x_max = X[:, 0].min() - 0.5, X[:, 0].max() + 0.5
 y_min, y_max = X[:, 1].min() - 0.5, X[:, 1].max() + 0.5
 
+
+# %%
+# Scatter Plot of the Iris dataset
+# --------------------------------
```

Comment:
In the section below, we can use the following code. I don't find it a good practice to not have the ticks and numbers:

```python
_, ax = plt.subplots()
scatter = ax.scatter(X[:, 0], X[:, 1], c=y, cmap=plt.cm.Set1, edgecolor="k")
ax.set(xlabel=iris.feature_names[0], ylabel=iris.feature_names[1])
_ = ax.legend(
    scatter.legend_elements()[0], iris.target_names, loc="lower right", title="Classes"
)
```